client/src/components/Reviews/components/ReviewList.jsx

```diff
@@ -30,8 +31,19 @@ const MoreReviewsButton = styled.button`
   bottom:  2%;
   cursor: pointer;
 `;
+const StyledSelect = styled.select`
+  border: none;
+`;
 
 function ReviewList() {
+  const [currentSort, setCurrentSort] = useState('relevant');
+  useEffect(() => {
+    // TODO axios.get() with currentSort param
+  }, [currentSort]);
+  function HandleSort(e) {
+    e.preventDefault();
```

Comment:
Not sure that `.preventDefault()` is necessary here. It's good to use on submit events to prevent page refresh, but I don't think that the change event on a select element impacts the page.